code/modules/research/designs/medical_designs.dm

```diff
@@ -10,12 +10,22 @@
 	name = "Biotech device prototype ([item_name])"
 
 /datum/design/item/health_analyzer
+	name = "Health Analyzer"
```

Comment:
This should be a subclass of medical designs, with a lowercase name.

Reply:
Donezo. It's still camelcase because that fits with the other medical items in the research designs doc.